SpacemanDMM.toml

```diff
@@ -5,5 +5,8 @@ dreamchecker = true
 disallow_relative_type_definitions = true
 disallow_relative_proc_definitions = true
 
+[diagnostics]
+var_in_proc_parameter = false
```

Comment:
```suggestion
var_in_proc_parameter = "off"
```

It's expecting an error type, which is a string.

Reply:
https://github.com/SpaceManiac/SpacemanDMM/blob/8c9b6b19c1d68b23d6f1c4248dee83b7546da2cb/crates/dreammaker/src/parser.rs#L1054

Reply:
Well that's misleading: 
![image](https://user-images.githubusercontent.com/5714543/144699645-73e685bf-1240-4c87-8774-7d6843ea89c2.png)